include/Frame.h

```diff
@@ -73,17 +73,17 @@ namespace openshot
 	 * There are many ways to create an instance of an openshot::Frame:
 	 * @code
 	 *
-	 * // Most basic: a blank frame (300x200 blank image, 48kHz audio silence)
+	 * // Most basic: a blank frame (1x1 blank image, 44.2kHz audio silence)
```

Comment:
```suggestion
	 * // Most basic: a blank frame (1x1 blank image, 44.1kHz audio silence)
```

All of those `44.2`s should be `44.1`.